app/jenkins-for-jira-ui/src/common/analytics/analytics-client.ts

```diff
@@ -21,13 +22,6 @@ interface TenantType {
 	TRELLO_WORKSPACE_ID: string
 }
 
-enum EnvType {
```

Comment:
Moved this out into a separate file to emulate the same behaviour on front/back ends